tests/zfs-tests/tests/functional/history/history_005_neg.ksh

```diff
@@ -42,9 +42,9 @@
 #		zpool iostat
 #
 # STRATEGY:
-#	1. Create a test pool.
```

Comment:
Why remove the period here (and in 3)?

Reply:
Just to have all three the same. Not one with full stop and other not.